Firmware/Marlin_main.cpp

```diff
@@ -10946,7 +10946,7 @@ void restore_print_from_eeprom(bool mbl_was_active) {
     enquecommand(cmd);
 
   // Unretract.
-    sprintf_P(cmd, PSTR("G1 E%0.3f F2700"), default_retraction);
+    sprintf_P(cmd, G1_E_F2700, default_retraction);
```

Comment:
Note there is one change here. There was a missing `"-"` modifier. It just makes sure the E value is left adjusted.